src/main/java/mods/railcraft/common/core/RailcraftConfig.java

```diff
@@ -262,7 +264,7 @@ private static void loadBlockTweaks() {
     }
 
     private static void loadItemTweaks() {
-//        trackingAuraEnabled = get(CAT_TWEAKS_ITEMS + ".goggles", "trackingAura", true, "Change to '{t}=false' to disable the Tracking Aura");
+        trackingAuraEnabled = get(CAT_AURAS + ".goggles", "trackingAura", true, "Change to '{t}=false' to disable the Tracking Aura");
```

Comment:
I think this config entry is useful but I don't know what category does it belong to. It controls whether the aura is enabled and is used in code.